src/main/java/exception/OutOfBoundIndexException.java

```diff
@@ -0,0 +1,14 @@
+package exception;
+
+/**
+ * This exception is thrown when user is trying to access a task with an out of bound index.
+ */
+public class OutOfBoundIndexException extends DukeException {
```

Comment:
You could start the class name with "Duke...." as in you could have a DukeOutOfBoundIndexException. This could be done for all other classes extending DukeException.